traversal/planner/GraphPlanner.java

```diff
@@ -163,15 +153,11 @@ private boolean isOptimal() {
     }
 
     private boolean isError() {
-        return resultStatus == INFEASIBLE || resultStatus == UNBOUNDED || resultStatus == ABNORMAL;
-    }
-
-    MPSolver solver() {
-        return solver;
+        return resultStatus == ERROR;
     }
 
-    MPObjective objective() {
-        return solver.objective();
+    Optimiser solver() {
+        return optimiser;
```

Comment:
conflicted as to whether we should have `//traversal/optimiser/Optimiser` vs `//traversal/solver/Solver` vs `//traversal/optimiser/Solver` !